### DISCUSSION TITLE
Is it possible to forward logs to fluentd or logstash or promtail ?

### DISCUSSION THREAD

**Discussion by pingping95** [loki4j/loki-logback-appender#176] [Q&A]
Hi.

I`m using Loki logback appender and push logs to Loki directly.

But I want to drop unneeded logs so that Loki does not store unneeded logs from SpringBoot.

So, Is it possible to forward logs to fluentd or logstash or promtail ?

Thanks!

**Comment by nehaev** [comment]
Hi @pingping95,

Loki4j is for sending logs directly to Loki. Probably you can filter unneeded logs using logback's [filters](https://logback.qos.ch/manual/filters.html)

If you need  to forward logs to fluentd or logstash or promtail , you can simply use other logback appenders for that (console appender forfluentd and  promtail, and logstash appender for logstash).
